theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/CurveEditorPopover.tsx

```diff
@@ -352,9 +359,14 @@ const CurveEditorPopover: React.FC<IProps> = (props) => {
             tabIndex={0}
             onKeyDown={onEasingOptionKeydown(preset)}
             ref={optionsRef.current[preset.label]}
+            onMouseOver={() => {
+              isFilterUpdatePrevented.current = true
+              editorState.hoverValue(preset.value)
+            }}
+            onMouseOut={() => editorState.discardHoverValue()}
             onClick={() => {
               setHighlightedIndex(displayedPresets.indexOf(preset))
-              editorState.temporarilySetValue(preset.value)
+              editorState.setValue(preset.value)
               //props.onRequestClose()
```

Comment:
Change: Delete commented code or document it's purpose.

Reply:
Deleted